front/app/utils/actionTakingRules.ts

```diff
@@ -146,7 +146,8 @@ export const getIdeaPostingRules = ({
 
     if (
       signedIn &&
-      (isAdmin({ data: authUser }) || isProjectModerator({ data: authUser }))
+      (isAdmin({ data: authUser }) ||
```

Comment:
Fixed this related bug too - moderators were able to try and submit multiple surveys on projects they were not moderators on - it failed, but now made sure the button is disabled in this situation

Reply:
Do you think we should keep it in, but add the projectId instead so moderators can still submit multiple surveys responses? I think we have it that way for admins so they can test the resident experience :thinking: 

```
 isProjectModerator({ data: authUser }, project.data.id))
```